docs/guide/installation/production-setup.md

```diff
@@ -312,7 +323,7 @@ Please find the key sections of the `vue-storefront/config/local.json` file desc
 },
 ```
 
-We're setting up the product's endpoint to https://prod.vuestorefront.io/api/catalog (please use your domain accordingly of course). As you may notice, the `/api` url is proxied by the NGINX to `localhost:8080` - our `vue-storefront-api` instance.
+We're setting up the product's endpoint to https://prod.vuestorefront.io/api/catalog (please use your domain accordingly of course). As you may notice, the `https://prod.vuestorefront.io/api` url is proxied by the NGINX to `localhost:8080` - our `vue-storefront-api` instance.
```

Comment:
We're setting up the product's endpoint => We're setting up the catalog endpoint